cps/main.c

```diff
@@ -377,9 +377,9 @@ process_egress(struct cps_config *cps_conf, struct gatekeeper_if *iface,
 	num_tx = rte_eth_tx_burst(iface->id, tx_queue,
 		forward_bufs, num_forward);
 	if (unlikely(num_tx < num_forward)) {
-		uint16_t i;
-		for (i = num_tx; i < num_forward; i++)
-			rte_pktmbuf_free(forward_bufs[i]);
+		uint16_t j;
+		for (j = num_tx; j < num_forward; j++)
+			rte_pktmbuf_free(forward_bufs[j]);
```

Comment:
A simpler solution is to drop the line `uint16_t i;` since variable `i` is already available here.